airbyte-integrations/connectors/source-hubspot/components.py

```diff
@@ -426,7 +432,7 @@ def convert_datetime_string_to_ab_datetime(datetime_str: str) -> Optional[Airbyt
         try:
             return ab_datetime_parse(datetime_str)
         except (ValueError, TypeError, OverflowError) as ex:
-            logger.warning(f"Couldn't parse date/datetime string field. Timestamp field value: {datetime_str}. Ex: {ex}")
+            pass
```

Comment:
this is unrelated to the actual fix, but our logs for some cases are extremely noisy because this error actually comes up quite frequently depending on customer configuration.

I left the second warning because if we truly can't parse it even after we got rid of the milliseconds part, we probably want and indication of that happening.